js/apps/filter-page/controllers/filterPageCtrl.js

```diff
@@ -1,25 +1,25 @@
 ((app) => {
 	'use strict';
 
-	const FilterPageCtrl = function FilterPageCtrl($scope, cardService, filterService, $animate, $timeout, CONSTANTS) {
-		const self = this;
+	const FilterPageCtrl = function FilterPageCtrl($location, $scope, cardService, filterService, $animate, $timeout, CONSTANTS) {
+		const vm = this;
```

Comment:
What does `vm` mean in this context?

Reply:
Oh wait, I think I got it.  Might want to spell out `viewModel`.

Reply:
yea, it's kinda the defactor standard we have been using for `viewModel` in angular